packages/form-js-editor/test/spec/form.json

```diff
@@ -3,12 +3,12 @@
   "type": "default",
   "components": [
     {
-      "id": "Field_1",
+      "id": "Text_1",

```

Comment:
Do we generate semantic IDs again? If so, why and where do we do that?

Reply:
We don't. This is meant to make testing easier by using semantic IDs in the test JSON.

Reply:
We can revert this if you don't like it.

Reply:
It is O.K.

Just wanted to make sure I did not miss a thing.